lib/hyrax/transactions/apply_change_set.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+require 'hyrax/transactions/transaction'
+
+module Hyrax
+  module Transactions
+    ##
+    # Applies and saves a `ChangeSet`.
+    #
+    # This transaction is intended to ensure appropriate results for a Hyrax
+    # model when saving changes from a `ChangeSet`. For example: it will set the
+    # system-managed metadata like modified date.
+    #
+    # If your application has custom system managed metadata, this is an
+    # appropriate place to inject that behavior.
+    #
+    # This will also validate the `ChangeSet`. Which validations to use is
+    # delegated on the `ChangeSet` itself.
+    #
+    # @since 3.0.0
+    #
+    # @example Applying a ChangeSet to a Work
+    #   work = Hyrax::Work.new
+    #   change_set = Hyrax::ChangeSet.for(work)
+    #   change_set.title = ['Comet in Moominland']
+    #
+    #   transaction = Hyrax::Transactions::ApplyChangeSet.new
+    #   result = transaction.call(change_set)
```

Comment:
It would be nice to see what the data type of this result is. It SEEMS like it might be a resource? But it would be nice to see here in the documentation.

Reply:
@straleyb it's definitely not a resource, so i guess that means there needs to be documentation here. https://github.com/samvera/hyrax/pull/4244/files/2cd191b9d812382eb7a2ecd580809ecaeb5e769c#r374209760

note that the `#call` documentation *does* give an explicit return type, and there's more narrative about this there and in the top-level `Transaction` class documentation.

Reply:
I think im understanding that the result is either a Success or Failure. Still would like to see that in the documentation here though

Reply:
sorry, does the suggested edit satisfy this request? or is there more needed?